app/scripts/lib/experiments/grouping-rules/disabled-button-state.js

```diff
@@ -30,15 +30,7 @@ define((require, exports, module) => {
         return false;
       }
 
-      let choice;
-
-      if (subject.forceExperiment === this.name && subject.forceExperimentGroup) {
```

Comment:
Since this logic is now taken care of in index.js, no need for each experiment to duplicate it.

Reply:
👍